src/modules/newTask_handler.js

```diff
@@ -0,0 +1,26 @@
+import { compileTaskItem } from './task_UI_Manager.js';
+import { arrTasks, Task, addTask } from './task_DatabaseMS.js';
+import { addIconEvent, addInputDescEvent } from './event_handlers.js';
+
+const newTaskListener = () => {
+  const textInput = document.getElementById('new-task');
+  textInput.addEventListener('change', (event) => {
+    event.preventDefault();
+    if (textInput.value.trim() === '') {
+      alert('Sorry! Task Cannot be Empty \n Tasks Not added');
+    } else {
+      addTask(Task(textInput.value.trim()));
+      const ulTasks = document.getElementById('ul-tasks');
+      const newTaskAdded = arrTasks[arrTasks.length - 1];
+      ulTasks.appendChild(compileTaskItem(newTaskAdded));
+      addIconEvent(newTaskAdded.index);
+      addInputDescEvent(newTaskAdded.index);
+      // addCheckBoxEvent(arrTasks[i].index);
```

Comment:
[ ] Kindly remove commented code from your project, This will make sure your project code is clean. You can leave comments about your code but not commented code It's not a good practice.